cmd/app/cmd.go

```diff
@@ -247,7 +247,7 @@ func gatherCredentials(flags *cmdFlags, config *configuration.Config) []*Credent
 		}
 	}
 
-	if len(flags.ghOAuthToken) > 0 {
+	if len(flags.ghOAuthToken) >= 0 {
```

Comment:
I think that you can completely remove this flag because `len` function will always return a number >= 0

Reply:
I agree